app/views/catalog/facet.html.erb

```diff
@@ -12,13 +12,21 @@
 
 </div>
 <div class="modal-body">
+  <div role="region" id="facet_list_header" aria-live="polite">
+    <% if @pagination.prefix && @display_facet.index? %>
+      <h4>Topics starting with "<%= @pagination.prefix %>" </h4>
```

Comment:
Change `Topics` to something that holds the facet title, `@display_facet` maybe?